plugins/languages/cmake-tools.nix

```diff
@@ -0,0 +1,134 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "cmake-tools";
+  originalName = "cmake-tools.nvim";
+  defaultPackage = pkgs.vimPlugins.cmake-tools-nvim;
+
+  maintainers = [ helpers.maintainers.NathanFelber ];
+
+  extraPlugins = with pkgs.vimPlugins; [ plenary-nvim ];
```

Comment:
This should be dealt with at the nixpkgs level.

Reply:
The dependency on `plenary.nvim` you mean? I can have a look when I get some time.

Reply:
https://github.com/NixOS/nixpkgs/pull/328534

Reply:
We should have https://github.com/NixOS/nixpkgs/pull/328534 in nixos-unstable within a day or two and our CI should also be back online soon.

Once we've updated our lockfile you'll be able to rebase and remove this line. I think we can wait for that before merging.